plugins/admin/README.adoc

```diff
@@ -152,7 +153,8 @@ Unset Knative route domain mydomain.com
 $ kn admin registry add \
   --server=[REGISTRY_SERVER_URL] \
   --username=[REGISTRY_USER] \
-  --password=[REGISTRY_PASSWORD]
+  --password=[REGISTRY_PASSWORD] \
+  -n [NAMESPACE]
```

Comment:
this is harmless, but maybe we can use the long name in the readme

Reply:
Thank you for your review! Based on the format problem I generally modified all of the `registry` command outputs to sync with `kn` and also added `serviceaccount` flag. Please review again and leave any comments.